driver-mct/shr/seq_infodata_mod.F90

```diff
@@ -451,7 +451,7 @@ SUBROUTINE seq_infodata_Init( infodata, nmlfile, ID, pioid, cpl_tag)
          wv_sat_scheme, wv_sat_transition_start,           &
          wv_sat_use_tables, wv_sat_table_spacing,          &
          tfreeze_option, glc_renormalize_smb,              &
-         ice_gnam, rof_gnam, glc_gnam, wav_gnam,           &
+         ice_gnam, rof_gnam, glc_gnam, wav_gnam,      &
```

Comment:
We probably don't want these whitespace changes

Reply:
yes this is left over from my first attempt/method at flexible spectra  - I will be removing them